prefs.ui

```diff
@@ -2,6 +2,18 @@
 <interface domain="auto-activities">
   <object class="AdwPreferencesGroup" id="MainWidget">
     <property name="title" translatable="yes">Preferences</property>
+    <child>
+      <object class="AdwActionRow">
+        <property name="title" translatable="yes">Show Apps</property>
```

Comment:
It would be interesting to add a subtitle with something like "Show apps grid" or something like that. I think saying "Show Apps" might be a bit confusing.